tools/aws_cli.py

```diff
@@ -155,6 +156,17 @@ def StartAnalysisVm(args: 'argparse.Namespace') -> None:
         'Created key pair {0:s} in AWS. Your private key is saved in: '
         '{1:s}'.format(key_name, path))
 
+  # Load up the startup script
+  startup_script = None
```

Comment:
Since `startup_script` is assigned in both clauses of the `if` block, this can be removed